From fc2854068e772b8aa411da86787249b5ab064e17 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:49:29 -0400
Subject: [tools] lib/traceevent: Harmonize the install messages in lib-traceevent

Message-id: <1408953210-26343-82-git-send-email-jolsa@redhat.com>
Patchwork-id: 88774
O-Subject: [PATCH RHEL7.1 BZ1133083 081/322] tools lib traceevent: Harmonize the install messages in lib-traceevent
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit f33c5cd73172c4fb81daa83a4f6d322210172fa0
Author: Jiri Olsa <jolsa@redhat.com>
Date: Tue Dec 3 14:09:20 2013 +0100

description
===========
Removing the 'to ...' part out of the install message, because it does
not fit to the rest of the build messages we use.

Before:
  INSTALL  plugin_hrtimer.so    to      /home/jolsa/libexec/perf-core/traceevent/plugins
  INSTALL  plugin_jbd2.so       to      /home/jolsa/libexec/perf-core/traceevent/plugins
  INSTALL  plugin_kmem.so       to      /home/jolsa/libexec/perf-core/traceevent/plugins
  INSTALL  plugin_kvm.so        to      /home/jolsa/libexec/perf-core/traceevent/plugins
  INSTALL  plugin_mac80211.so   to      /home/jolsa/libexec/perf-core/traceevent/plugins
  INSTALL  plugin_sched_switch.so       to      /home/jolsa/libexec/perf-core/traceevent/plugins
  INSTALL  plugin_function.so   to      /home/jolsa/libexec/perf-core/traceevent/plugins
  INSTALL  plugin_xen.so        to      /home/jolsa/libexec/perf-core/traceevent/plugins
  INSTALL  plugin_scsi.so       to      /home/jolsa/libexec/perf-core/traceevent/plugins

Now:
  INSTALL  plugin_jbd2.so
  INSTALL  plugin_hrtimer.so
  INSTALL  plugin_kmem.so
  INSTALL  plugin_kvm.so
  INSTALL  plugin_mac80211.so
  INSTALL  plugin_sched_switch.so
  INSTALL  plugin_function.so
  INSTALL  plugin_xen.so
  INSTALL  plugin_scsi.so
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/lib/traceevent/Makefile b/tools/lib/traceevent/Makefile
index 1526798..f946851 100644
--- a/tools/lib/traceevent/Makefile
+++ b/tools/lib/traceevent/Makefile
@@ -166,7 +166,7 @@ else
   print_plugin_obj_compile = echo '  CC FPIC  '$(OBJ);
   print_plugin_build =  echo '  BUILD    PLUGIN '$(OBJ);
   print_static_lib_build = echo '  BUILD    STATIC LIB '$(OBJ);
-  print_install =  echo '  INSTALL  '$1' to $(DESTDIR_SQ)$2';
+  print_install =  echo '  INSTALL  '$1;
 endif
 
 do_fpic_compile =     \
-- 
1.7.1